### PR TITLE
Implement ExperimentsDebugActivity command to switch between Kinto instances.

### PR DESCRIPTION
**Pull request by travis79**
Using the ExperimentsDebugActivity, implement the "setKintoInstance" command to switch between "dev", "staging", or "prod" instances. 

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

**Comment by travis79** [comment]
@georgf I wanted to get this started, I know I'll be gone next week so feel free to run with this if you need it landed next week.

File: components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt [hunk 1]
```diff
@@ -38,6 +41,21 @@ class ExperimentsDebugActivity : Activity() {
             }
         }
 
+        if (intent.hasExtra(SET_KINTO_INSTANCE_EXTRA_KEY)) {
+            val instance = intent.getStringExtra(SET_KINTO_INSTANCE_EXTRA_KEY)
+
+            val config: Configuration? = when (instance) {
+                "dev" -> Configuration(kintoEndpoint = ExperimentsUpdater.KINTO_ENDPOINT_DEV)
```
Comment (Dexterp37) [inline components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt]: Note that `Configuration()` will drop any custom value that was previously provided as part of the configuration (e.g. the http client), and fall back to the defaults. What you really want here is to do `Configuration.copy(kintoEndpoint = ...)`

**Comment by georgf** [comment]
One note here: i think we'll need to persist the kinto instance choice between restarts.
Otherwise it will be hard to test behavior across restarts (e.g. start experiment, stop app, delete experiment on server, restart app, verify experiment is stopped).

File: components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt (hunk 1 above)
Comment (travis79) [reply components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt]: Thanks, I will fix this as I get back to work on this PR!

**Comment by travis79** [comment]
> One note here: i think we'll need to persist the kinto instance choice between restarts.
> Otherwise it will be hard to test behavior across restarts (e.g. start experiment, stop app, delete experiment on server, restart app, verify experiment is stopped).

That makes sense, I'll add persistence to it, but we may need to handle it a little differently than the other configuration items then.

**Comment by travis79** [comment]
On second thought, in thinking about persistence...   do we really need to persist this when the app can just be restarted using the same kinto instance?  It seems to me like this would make sense when being run from a test as this is intended to be.  If we allow for persistence, then it seems possible to accidentally override the kinto instance to something unintended if someone didn't reset it back to the proper one.  I guess what I'm trying to say is, I would rather have the extra step be in launching the app to get it changed rather than trying to remember to reset the instance back to production every time.

**Comment by georgf** [comment]
> On second thought, in thinking about persistence... do we really need to persist this when the app can just be restarted using the debug activity to set it to the correct instance? It seems to me like this would make sense when being run from a test as this is intended to be. If we allow for persistence, then it seems possible to accidentally override the kinto instance to something unintended if someone didn't reset it back to the proper one. I guess what I'm trying to say is, I would rather have the extra step be in launching the app to get it changed rather than trying to remember to reset the instance back to production every time.

That's a good point. For now, let's simply require launching the app with the debug command to change kinto endpoints.
We can iterate later on this if needed.

File: components/service/experiments/README.md [hunk 2]
```diff
@@ -73,12 +73,15 @@ In the above:
     |key|type|description|
     |---|----|-----------|
     | updateExperiments | boolean (--ez) | forces the experiments updater to run and fetch experiments immediately |
+    | setKintoInstance | string (--es) | sets the Kinto instance to the specified instance ("dev", "staging", "prod" only) |
 
-For example, to direct a release build of the Glean sample application to update experiments immediately, the following command
-can be used:
+For example, to direct a release build of the glean sample application to (1) update experiments immediately and
```
Comment (Dexterp37) [inline components/service/experiments/README.md]: nit: we settled on `Glean` rather than `glean` :)

Comment (travis79) [reply components/service/experiments/README.md]: oops ;)

Comment (travis79) [reply components/service/experiments/README.md]: merged rolled right over it...

**Pull request merged**